src/main/java/storages/CustomArrayList.java

```diff
@@ -0,0 +1,86 @@
+package storages;
```

Comment:
add examples of how to use each method of the storages into your main method

Reply:
Fixed in https://github.com/kolosokvit/ticket-service/pull/3/commits/5d4f5d14b38c811869c54cba7df89aa28f97c409